store/src/java/com/zimbra/cs/service/account/Auth.java

```diff
@@ -446,24 +485,9 @@ private Element doResponse(Element request, AuthToken at, ZimbraSoapContext zsc,
         String skin = SkinUtil.chooseSkin(acct, requestedSkin);
         ZimbraLog.webclient.debug("chooseSkin() returned "+skin );
         if (skin != null) {
-            response.addElement(AccountConstants.E_SKIN).setText(skin);
+            response.addUniqueElement(AccountConstants.E_SKIN).setText(skin);
```

Comment:
reverted to addElement, only reason for change was that addElement has been deprecated.